lua/lspinstall/servers/terraform.lua

```diff
@@ -0,0 +1,48 @@
+local config = require'lspconfig'.terraformls.document_config
+require'lspconfig/configs'.terraformls = nil -- important, immediately unset the loaded config again
+config.default_config.cmd[1] = "./terraform-ls"
+
+return vim.tbl_extend('error', config, {
+  install_script = [[
+  os=$(uname -s | tr "[:upper:]" "[:lower:]")
+  arch=$(uname -m | tr "[:upper:]" "[:lower:]")
+
+  case $os in
+  linux)
+    platform="linux"
+  ;;
+  darwin)
+    platform="darwin"
+  ;;
+  windows)
```

Comment:
Could you remove Windows again? Windows is not supported, it can't even run this script. And if it is Windows WSL, `$os` will say linux anyways.

Reply:
OK removing windows

Reply:
Removed